include/vcpkg/base/messages.h

```diff
@@ -1073,7 +1089,15 @@ namespace vcpkg
                     "calling {system_api} failed with {exit_code} ({error_msg})");
     DECLARE_MESSAGE(ToolFetchFailed, (msg::tool_name), "", "Could not fetch {tool_name}.");
     DECLARE_MESSAGE(ToolInWin10, (), "", "This utility is bundled with Windows 10 or later.");
+    DECLARE_MESSAGE(UnableToExtractArchive,
+                    (msg::package_name, msg::path),
+                    "",
+                    "Unable to extract '{package_name}' archive from Installer '{path}'.");
     DECLARE_MESSAGE(UnexpectedErrorDuringBulkDownload, (), "", "an unexpected error occurred during bulk download.");
+    DECLARE_MESSAGE(UnexpectedExtension,
+                    (msg::value),
+                    "'{value}' is the archive extension.",
```

Comment:
You added `{extension}` so you should use it here. Should this one be combined with `MissingExtension` somehow?